dxr/app.py

```diff
@@ -249,6 +251,23 @@ def raw(tree, path):
     return send_file(data_file, mimetype=guess_type(path)[0])
 
 
+@dxr_blueprint.route('/<tree>/rawrev/<revision>/<path:path>')
```

Comment:
I have to admit I was unaware of the dashes-vs-underscores debate, but even had I been, we do use dashes in filters as you point out, so dashes are more consistent - thanks!

Yes, the new endpoint is to serve up just the plain raw binary of a file at a particular revision rather than embedded in the dxr layout.  Then the /rev/ page for a binary image can have a /raw-rev/ img href to the image at that rev (so the rev image gets displayed on its /rev/ page), and for svg /rev/ pages we can put a "view image" link in the nav pane to the svg served up as svg by a /raw-rev/.